transactions/read-only/parallel.md

```diff
@@ -23,112 +23,115 @@ Chain APIs can be classified into reads and writes.
+| set_whitelist_blacklist                    | medium_high | main thread:controller blacklist, whitelist|  no      | no                       |
+| get_integrity_hash                         | medium_high | calls abort_block() and schedule_production_loop() | no | no                     |
+| create_snapshot                            | medium_high | calls abort_block() and schedule_production_loop() | no | no                     |
+| schedule_snapshot                          | medium_high | snapshot_request_information | yes                    | no                       |
+| get_snapshot_requests                      | medium_high |                              | yes                    | yes                      |
+| unschedule_snapshot                        | medium_high | snapshot_request_information | yes                    | no                       |
+| get_scheduled_protocol_feature_activations | medium_high |                              | yes                    | yes                      |
+| schedule_protocol_feature_activations      | medium_high |                              | no                     | no                       |
+| get_supported_protocol_features            | medium_high |                              | yes                    | yes                      |
+| get_account_ram_corrections                | medium_high |                              | yes                    | yes                      |
+| get_unapplied_transactions                 | medium_high |                              | yes                    | yes                      |
 
 ### Net APIs
 Net APIs do modify state but do not modify the state accessible to read-only transaction execution. They are received on the HTTP thread, processed on the main thread, and responses are sent on the HTTP thread.
 
-| API                         | priority    | Global data modified           | Safe to read-only trx? |
-|-----------------------------|-------------|--------------------------------|------------------------|
-| connect                     | medium_high | main thread: net::connections  | yes                    |
-| disconnect                  | medium_high | main thread: net::connections  | yes                    |
-| status                      | medium_high | none                           | yes                    |
-| connections                 | medium_high | none                           | yes                    |
+| API                         | priority    | Global data modified           | Safe to read-only trx? | Safe to run in parallel? |
+|-----------------------------|-------------|--------------------------------|------------------------|--------------------------|
+| connect                     | medium_high | main thread: net::connections  | yes                    | no                       |
+| disconnect                  | medium_high | main thread: net::connections  | yes                    | no                       |
```

Comment:
connect/disconnect are both yes. They are protected by mutexes and can run in parallel from any thread.